src/Options/index.js

```diff
@@ -196,6 +196,9 @@ export interface ParseServerOptions {
   cacheAdapter: ?Adapter<CacheAdapter>;
   /* Adapter module for email sending */
   emailAdapter: ?Adapter<MailAdapter>;
+  /* Whether Parse Pointers should be encoded in Cloud Code.
+  :DEFAULT: false */
+  encodeCloudPointers: ?boolean;
```

Comment:
Pleas take a look whether this makes sense and feel free to correct where necessary:

```suggestion
  /* If set to `true`, a `Parse.Pointer` that is in the payload when calling a Cloud Function will be converted to an instance of a `Parse.Pointer`. If `false`, the object will not be converted and instead be a plain JavaScript object, which contains the raw data of a `Parse.Pointer` but is not an actual instance of a `Parse.Pointer`. Default is `false`. <br><br>ℹ️ The expected behavior would be that the object is converted to an instance of `Parse.Pointer`, so you would normally set this option to `true`. The default is `false` because this is a temporary option that has been introduced to avoid a breaking change when fixing a bug where `Parse.Pointer` objects are not converted to actual instances of `Parse.Pointer`.
  :DEFAULT: false */
  encodePointerInCloudFunction: ?boolean;
```